fido/fido.py

```diff
@@ -477,7 +477,7 @@ def identify_stream(self, stream, filename):
             if (os.name != "nt"):
                 try:
                     self.current_file = os.readlink("/proc/self/fd/0")
-                except:
+                except:  # noqa: E722
```

Comment:
Similarly, it looks like `OSError` is the only exception that can happen on both platforms - aside from maybe a weird unicode decode error on 3, but if that happens I think we'd want to know about it.

Reply:
Thank you!!!!